controllers/solrcloud_controller_test.go

```diff
@@ -38,6 +39,11 @@ func newBoolPtr(value bool) *bool {
 	return &newBool
 }
 
+func newIntPtr(value int64) *int64 {
```

Comment:
You should be able to use `pointer.Int64(...)` for this. (The pointer library being "k8s.io/utils/pointer")

Reply:
I see the I probably added the methods above that do the same thing. So we can always go and remove them independently, but let's not add another one.